ansible_runner_service.py

```diff
@@ -156,13 +185,30 @@ def main(test_mode=False):
         app.config['WTF_CSRF_ENABLED'] = False
         return app.test_client()
 
-    # Start the API server
-    app.run(host=configuration.settings.ip_address,
-            port=configuration.settings.port,
-            threaded=True,
-            ssl_context=ssl_context,
-            debug=configuration.settings.debug,
-            use_reloader=False)
+    if configuration.settings.mode == 'prod':
+        cancel_remove_artifacts_thread = threading.Event()
```

Comment:
I will put all the "old artifacts removal" stuff in one procedure ... for example "clean_old_artifacts".
 And if we use "artifacts_remove_age=0" to disable this feature, we can do inside this procedure 
```
def clean_old_artifacts():
    if configuration.settings.artifacts_remove_age>0:
        cancel_remove_artifacts_thread = threading.Event()
        ....

```